app/controllers/events_controller.rb

```diff
@@ -76,19 +66,8 @@ def event_params
     params.require(:event).permit(:title, :description, :location, :date)
   end
 
-  # This is the check to allow the authorisation of a user's access
-  def logged_in_user
-    unless logged_in?
-      flash[:danger] = "Please log in."
-      redirect_to login_url
-    end
-  end
-
-  def correct_user
-    @event = Event.find(params[:id])
-    @user = User.find(@event.host_id)
-    redirect_to(current_user) unless @user == current_user
-    # this sends a user to their own page if they try to access a different profile
+  def find_event
```

Comment:
if you call this method simply `event`, you will be able to simply use it when needed, e.g. in update:

```
def update
     if event.update_attributes(event_params)
       flash[:success] = "Event updated"
       redirect_to event
     else
       render 'edit'
     end
   end
```